### PR TITLE
Add Actions In All Controllers

### PR DESCRIPTION
**Pull request by samiullahbahadur**
Hi Moise 
### I have did the below tasks:

1. I have change the user migrate( we need to add authenticate through react/redux instead of devise )
2. I have create all necessary actions in user controller
3. I have create all necessary actions in doctor controller  
4. I have create all necessary  actions in reservation controller 
5. I have change routes

### CODE REVIEW

File: app/controllers/doctors_controller.rb [hunk 1]
```diff
@@ -1,11 +1,50 @@
 class DoctorsController < ApplicationController
+  before_action :set_Doctor, only: [:edit, :update, :show]
```
Comment (moise-mulungu) [inline app/controllers/doctors_controller.rb]: The following method `set-Doctor` is undefined when I am trying to access this path.
I suggest you could write `set_doctor` instead.

**Review by moise-mulungu — CHANGES_REQUESTED** [review]
Good job @samiullahbahadur for one more progress for this milestone.
giphy (3)

You still have some changes to perform for the app to work perfectly:
- When I am trying to access a user page, there is an issue with rendering text/HTML format from the index action, please try to fix it.

**Review by moise-mulungu — APPROVED** [review]
Congratulations @samiullahbahadur on fixing the required changes:
- API :heavy_check_mark: 
- Database :heavy_check_mark: 
_There is nothing else to say ... than just to merge_
_Happy coding :computer:_

**Comment by kosher9** [comment]
Hi @moise-mulungu , @samiullahbahadur , I hope you are doing well. I am doing better now. I just pulled the project and went through it. 
You guys did a great job. 👏🏾 👏🏾 👏🏾 

But I noticed a few things we should avoid: 
- We should not merge a commit without all linters green checked. ✅ 
- The tasks were not well structured therefore some 'actions' we will not need were created.

I am going to try my best to make the kanban more exhaustive and put some security rules. 

Thank you for your understanding guys. Happy coding and see you soon.

**Pull request merged**